resources/js/dashboard/pages/Landing.vue

```diff
@@ -79,6 +81,13 @@ export default {
           name: 'administrator.epidemie'
         })
       }
+      if (this.user.roles.find((a) => a.name == EDIT_FORM || a.name == CREATE_FORM && a.name != ADMINISTRATOR)) {
```

Comment:
`if (this.user.roles.find((a) => (a.name == EDIT_FORM || a.name == CREATE_FORM) && a.name != ADMINISTRATOR)) {`